src/httpclient.c

```diff
@@ -429,6 +429,7 @@ h2o_httpclient_head_cb on_connect(h2o_httpclient_t *client, const char *errstr,
                                   h2o_httpclient_proceed_req_cb *proceed_req_cb, h2o_httpclient_properties_t *props,
                                   h2o_url_t *origin)
 {
+    usleep(req_delay*1000);
```

Comment:
Not a huge fan of this. Is there a better way of doing this?

I'd like to connect on a network and application level (h2/h3) but not proceed on I/O for a delay. This does not seem to work with `-i` as the function returns with `on_head` immediately.

Reply:
Found a working alternative for now.